src/wp-includes/option.php

```diff
@@ -1308,7 +1413,7 @@ function set_transient( $transient, $value, $expiration = 0 ) {
 		$transient_option  = '_transient_' . $transient;
 
 		if ( false === get_option( $transient_option ) ) {
-			$autoload = 'yes';
+			$autoload = 'yes'; // Consider using null, so default values are selected here.
```

Comment:
Please remove this comment, as that's not relevant for this ticket.
```suggestion
			$autoload = 'yes';
```